src/coord/src/catalog.rs

```diff
@@ -233,17 +240,29 @@ impl CatalogItem {
             CatalogItem::View(_) => "view",
             CatalogItem::Index(_) => "index",
             CatalogItem::Type(_) => "type",
+            CatalogItem::Func(_) => "func",
```

Comment:
I think this probably ought to be spelled out for when it appears in error messages. Actually we should probably just delete this entire function, and let people go through the `CatalogItemType`, which impls `fmt::Display` instead.